tests/test_hooks_folder.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from tests import BaseTestCase
+from hooks4git.scripts import get_hooks_path
+
+
+class HooksFolderTestCase(BaseTestCase):
+    def test_get_hooks_path_missing_folder_notcreate(self):
+        path = get_hooks_path("/tmp", create_if_missing=False)
```

Comment:
Why add `create_if_missing` parameter? You only call `get_hooks_path` with it at `True` and that's the behaviour one would one.

Same goes for `hooks_folder_name` if you don't need it in your current code, don't add it. It will just add unnecessary complexity.

Removing those parameters, 
* allow to remove variables which make code more difficult to understand, 
* remove conditional branching, making code simpler to read

Reply:
Ok, I have to agree. Ideally, I was not willing to touch the user's files or folders. So the create flag was somewhat expected in the future a default option to not create the folder unless the user added a --force option, or something. The idea was just to make a generic function, but yeah, as you said, unnecessary.

Reply:
Test cases were calling it with False to make, but not main code. If I eventually change my mind about the "--force" I will rethink about this workflow.